AdditionalCodingGuidelines.md

```diff
@@ -103,8 +103,13 @@ final String x = (condition) ? "value" : null;
 	 - The idea is to put focus the actual operations your code does instead of burying that
 		in the ceremony around it.
 
-
-8. Java Optionals.
+8. Usage of 'var'
+	Since Java 9, the usage of `var` is supported. This is still strongly typed, but notation is less verbose:
+	`String string = new String()` can become `var string = new String()`.
+	This can become less readable when using the return type of a method. To keep code as readable as possible, we only use 
+	this 'locally'. This means, a local value. Try to avoid using var for method return types, this makes code less readable 
```

Comment:
This is not clear. It reads as if the method return type can be specified as `var`, which is not true. And a local variable can also be assigned a method return value.

```suggestion
	this when the type is clear, such as a direct assignment of a string or scalar value or constructor. Try to avoid using `var` for variables assigned from method return values, this makes code less readable
```